src/components/Rocket.jsx

```diff
@@ -17,18 +17,16 @@ function Rockets() {
   return (
     <>
       <div>
-        {rockets.map(
-          ({ id, rocket_name, description, flickr_images, reserved }) => (
-            <div key={id}>
-              <img src={flickr_images} />
-              <h2>{rocket_name}</h2>
-              <p>{description}</p>
-              <Button onClick={() => bookHandler(id, reserved)}>
-                {reserved ? "Cancel Reservation" : "Reserve Rocket"}
-              </Button>
-            </div>
-          )
-        )}
+        {rockets.map(({ id, name, description, flickrImages, reserved }) => (
+          <div key={id}>
+            <img src={flickrImages} />
+            <h2>{name}</h2>
+            <p>{description}</p>
+            <Button onClick={() => bookHandler(id, reserved)}>
+              {reserved ? "Cancel Reservation" : "Reserve Rocket"}
+            </Button>
```

Comment:
Please kindly add a badge showing a Rocket is reserved when the user clicks on the reserve button, as it is the project's requirement. You can check [this](https://github.com/mertkantaroglu/react-redux-group-project/issues/25) for more details. 😄